jib-core/src/main/java/com/google/cloud/tools/jib/image/json/BadConfigurationFormatException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.image.json;
+
+/** Exception thrown when trying to parse a bad image configuration format. */
+public class BadConfigurationFormatException extends Exception {
```

Comment:
This exception as is seems fairly generic.  Would it be worth providing the Path or source object to the configuration file?

Reply:
Hmm, that's a good point, although I'm not sure what would be most useful to provide through this exception. Maybe I can just add a TODO here for now and we can change it if a good use-case pops up?

Reply:
Sure.